src/rabbit_channel.erl

```diff
@@ -411,6 +415,7 @@ handle_call({info, Items}, _From, State) ->
     end;
 
 handle_call(refresh_config, _From, State = #ch{virtual_host = VHost}) ->
+    % TODO: re-init interceptor_state?
```

Comment:
I think the interceptor state only needs to be updated when plugins are enabled/disabled